app/views/foods/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="flex items-center flex-col mt-48 mb-48 container mx-auto">
+  <%= link_to "New Food", new_food_path, class: 'btn' %>
+  <%if @foods.any?%>
+    <% @foods.each do |food| %>
+      <p>Name <%= food.name %></p>
+      <p>Measurement unit <%= food.measurement_unit %></p>
+      <p> Price <%= food.price %></p>
+      <%= button_to 'Delete', food_path(food), method: :delete %>
+    <% end %>
+  <%else%>
+    <div class="w-3/5 bg-white shadow-lg p-10">
+      <div>
+        <p>No Foods yet</p>
+      </div>
+    <div>
```

Comment:
- [ ] Hello team, Impressive job already 🙌. But my concern here is your design is not the same as the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/food_index.png?raw=true). Remember when working with a wire frame it is okay to update color and style but the layout should remain the same. 

### Your food list as of now 
![image](https://user-images.githubusercontent.com/69077061/219878453-f6c043c5-fbe6-494f-a6a2-9314264b298b.png)

Reply:
Thanks for the feedback.